backend/src/models/patient.js

```diff
@@ -27,10 +26,26 @@ const patientSchema = new Schema({
     gender:{
         type:String
     },
-    image:{
-        type:String,
-        default:""
+    isDiagnosed:{
+        type:Boolean,
+        default:false
+    },
+    patientresult:[ 
+        {
+        date:{
+            type:Date,
+    
+        },
+        glaucomatous:{
+            type:Boolean,
+            default:false
+    
+        },
+        image:{
+            type:String
```

Comment:
instead of glaucomatous, let's add "modelResult: { result: string, probability: number }" and "doctorFinalDecision: string", we can discuss more